java-squid/src/main/java/org/sonar/java/se/ConstraintManager.java

```diff
@@ -62,6 +65,14 @@ public SymbolicValue createSymbolicValue(Tree syntaxNode) {
       case INSTANCE_OF:
         result = new SymbolicValue.InstanceOfSymbolicValue(counter);
         break;
+      case MEMBER_SELECT:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1151)